db/migrate/20210312074713_add_provider_inputs.rb

```diff
@@ -0,0 +1,10 @@
+class AddProviderInputs < ActiveRecord::Migration[6.0]
+  def change
+    create_table :template_invocation_provider_input_values do |t|
```

Comment:
don't we want to simplify the table name? :thinking:

Reply:
Any suggestions?

Reply:
I think at least `invocation_provider_input_values` :thinking: but no hard feelings aganst this name.